util/check_dif_statuses.py

```diff
@@ -52,6 +52,14 @@ class _OTComponent(Enum):
     HW = 2
 
 
+class _DIFFunctionType(Enum):
+    """Type of DIF function."""
+    ALERT = 1
+    IRQ = 2
+    UNIMPLEMENTED = 3
+    OTHER = 4
```

Comment:
Is there a value in keeping this as a type?

Reply:
It is used in print_function_set() to determine the table header label.

Reply:
Sorry, I meant the `OTHER` literal which does not seem to be needed.

Reply:
Ah, gotcha. Fixed.